default/python/README.md

```diff
@@ -55,37 +55,12 @@ Add `default\python\` as content root.
 pip install -r default\python\requirements-dev.txt
 ```
 
-### Automate check with pre commit hooks
-Git could run a script
-([hook](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks)) 
-when you commit changes.  
-
-This feature could be used as duplication of the CI actions.
-The main difference are that hooks are optional 
-(a developer made a decision if they should be run) 
-and that hooks gives you feedback much faster 
-because they are run on developer machine.
-
-Hooks work on all operating systems.
-You need to install them once.
-Hook will track config file and update itself in case of changes. 
-
-- install git
-- install Python
-- [Install pre-commit](https://pre-commit.com/#install) 
-    ```sh
-    pip install pre-commit
-    ```
-- install hooks from the project root
-    ```sh
-    pre-commit install 
-    ```
-
-Notes:
-- First commit after installation/update of hooks may take some time.
-- You can skip checks by adding `-n, --no-verify` to `git commit`.
```

Comment:
Does the new "Installing pre-commit hooks" section really properly replace the above part? Just wondering...

Reply:
Ah ok. So you took the deleted part, rewrote it and put the rewrite in `BUILD.md`. It's not that I miss something specific, I just noticed that the rewrite looks quite different, and wasn't sure if the one really replaced the other, so I asked to be sure.